actions/knowledge/filehelper.ts

```diff
@@ -28,6 +30,28 @@ export async function getFileOrFolderSizeInKB(
   return 0;
 }
 
-export async function getBasename(filePath: string): Promise<string> {
-  return path.basename(filePath);
+export async function importFiles(
+  files: string[]
+): Promise<Map<string, FileDetail>> {
```

Comment:
nit: it looks like the return type of this method can be inferred automatically. It's usually a good idea in TS to use inferred return types so that the compiler can catch errors at compile time if anything changes inside of the function.